sdk/resourcemanager/azure-resourcemanager-network/src/test/java/com/azure/resourcemanager/network/VpnSiteTests.java

```diff
@@ -0,0 +1,297 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.network;
+
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.management.Region;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.resourcemanager.network.fluent.models.VirtualWanInner;
+import com.azure.resourcemanager.network.fluent.models.VpnSiteLinkInner;
+import com.azure.resourcemanager.network.models.DeviceProperties;
+import com.azure.resourcemanager.network.models.O365BreakOutCategoryPolicies;
+import com.azure.resourcemanager.network.models.O365PolicyProperties;
+import com.azure.resourcemanager.network.models.VpnLinkBgpSettings;
+import com.azure.resourcemanager.network.models.VpnLinkProviderProperties;
+import com.azure.resourcemanager.network.models.VpnSite;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+public class VpnSiteTests extends NetworkManagementTest {
+    private final Region region = Region.US_WEST;
+    private String vpnName = "";
+    private String vwName1 = "";
+    private String vwName2 = "";
+    private String vslName1 = "";
+    private String vslName2 = "";
+
+    @Override
+    protected void initializeClients(HttpPipeline httpPipeline, AzureProfile profile) {
+        vpnName = generateRandomResourceName("vpn", 8);
+        vwName1 = generateRandomResourceName("vw1", 8);
+        vwName2 = generateRandomResourceName("vw2", 8);
+        vslName1 = generateRandomResourceName("vsl1", 8);
+        vslName2 = generateRandomResourceName("vsl2", 8);
+        super.initializeClients(httpPipeline, profile);
+    }
+
+    private VirtualWanInner createVirtualWan(String vwName, Boolean disableVpnEncryption, String type) {
+        return networkManager.serviceClient()
+            .getVirtualWans()
+            .createOrUpdate(rgName, vwName,
+                new VirtualWanInner()
+                    .withLocation(region.name())
+                    .withDisableVpnEncryption(disableVpnEncryption)
+                    .withTypePropertiesType(type));
+    }
+
+    @Test
+    public void canCreateAndUpdateVpnSiteByMinimumParametersWithLinkIpAddress() {
```

Comment:
Would like to know what's the use case you want to cover here. Any official docs on these scenarios?

Reply:
Only one of the Link IP address/FQDN can be effective, and the Link IP address takes precedence.
Please following as : https://learn.microsoft.com/en-us/azure/virtual-wan/virtual-wan-site-to-site-portal#site